src/components/include/utils/threads/thread.h

```diff
@@ -216,6 +216,16 @@ class Thread {
     return handle_;
   }
 
+#if defined(QT_PORT)
```

Comment:
@dev-gh I think this setter is redundant. You're able to set `handle_` inside `threadFunc` even without it